### ISSUE TITLE
[v5]Export LAUNCH to PDF does not include the launch attributes

### ISSUE THREAD

**Issue by senkrish** [reportportal/reportportal#802]
**Describe the bug**
Exporting a launch to PDF is a great feature I love about ReportPortal. It currently includes:

- Launch Name
- Launch Description
- Owner
- Duration
- Launch Statistics

It does not seem to include the attributes I have saved for the launch. I have saved some key data regarding the launch as attributes (key:branch value:branch_name, key:timestamp value:date, etc). I do see the backend code to include the attributes in the exported data:

[I maybe reading the code wrong, but this is what I see]
PrintToPdf calls exportLauch(lauchId):
https://github.com/reportportal/service-api/blob/4576f2534309c0265d3881f5d5a10ace4191e3d1/src/main/java/com/epam/ta/reportportal/core/launch/impl/GetLaunchHandlerImpl.java#L251

This calls converParams(launchId) and includes the data in the PDF:
https://github.com/reportportal/service-api/blob/a7d27146e3574cafd314153d70f6ebea6d1f3670/src/main/java/com/epam/ta/reportportal/core/jasper/impl/LaunchJasperReportHandler.java#L68

But I do not see the attributes in the generated PDF. This behavior is the same in both HTML and XML export.

**To Reproduce**
- Create a launch with sample attributes (k1:v1, k2:v2)
- Export Launch to PDF using the hamburger icon next to Launch name.
- Notice missing attributes in the PDF

**Expected behavior**
Launch attributes should be included in the exported launch data, but it is not.

**Desktop (please complete the following information):**
 - OS: mac OSX (Mojave)
 - Browser: chrome
 - Version: 9.0

**Comment by evjlobanova** [comment]
@senkrish yes, you I right a templete doesnot have attributes, but this info is returned from back end.

We are planning to rework this feature in future.

But could you please describe your use case? Why do you need this exported report? How do you utilize them?

**Comment by senkrish** [comment]
The use case is that the PDF generated from a Launch is used by our Quality team to ensure all the verification and validation for a release is complete, and attach that as documentary evidence to the final release report.

The current template of the report is missing some vital data that would be useful for the Software Quality team:

- The branch/release that the Launch was run against (this is available in the attribute release:git_tag for all our launches)
- The exact date/time the testing was done (available in the attribute datetime:<UTC>)
- The commit sha of the HEAD of the branch (available in the attribute commit:<SHA>)

Including these in the template will add value to the Launch Report. I'm sure this situation is not unique to us.

Thanks for looking!

**Comment by senkrish** [comment]
@evjlobanova Can you provide an ETA for this enhancement? This is a blocker for rolling out RP more widely in our setup. Thanks.

**Comment by evjlobanova** [comment]
@senkrish This improvenent is not in our backlog, but i will discuss it internally.
We can not take in the nearest release, so that it is difficult to say the exact terms for improvement.
As a possible dissicion, i can propose you a paid support, in this case we can take this feature with priority number 1.

I will back latter, and tell you an approximate terms for realization.

**Comment by evjlobanova** [comment]
@DzmitryHumianiuk what do you think about it?

**Comment by DzmitryHumianiuk** [comment]
@evjlobanova worth of implementation
